pom.xml

```diff
@@ -68,31 +68,35 @@
 
       <plugin>
         <artifactId>maven-enforcer-plugin</artifactId>
-        <version>1.4.1</version>
+        <version>3.4.1</version>
         <executions>
           <execution>
             <id>enforce-java</id>
             <goals>
               <goal>enforce</goal>
             </goals>
-            <configuration>
-              <rules>
-                <!--
-                Ensure use of Java 8, as required by NXRM
-                -->
-                <requireJavaVersion>
-                  <version>[1.8,1.9)</version>
-                </requireJavaVersion>
-                <!--
-                Require Maven 3.3.3+
-                -->
-                <requireMavenVersion>
-                  <version>[3.3.3,)</version>
-                </requireMavenVersion>
-              </rules>
-            </configuration>
           </execution>
         </executions>
+        <configuration>
+          <rules>
+            <!--
+            Ensure use of Java 8, as required by NXRM
```

Comment:
Not certain about the Java rules going forward, but I think maybe Java 8 actually stops being supported after NXRM 3.70. See "Java" https://help.sonatype.com/en/sonatype-nexus-repository-system-requirements.html

This gets messy fast. 

Maybe add a link to the docs in the comment. At some point, we probably want to PREVENT the use of Java 8...

Reply:
This is just the JDK version required to build the plugin, not necessarily the same as the runtime requirement.

I’d be happy to raise this to 11 or 17. We could also clean up quite a few lines then.